src/main/java/seedu/address/model/entry/Location.java

```diff
@@ -0,0 +1,59 @@
+package seedu.address.model.entry;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a Event's Location in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidLocation(String)}
+ */
+public class Location {
+
+    public static final String MESSAGE_CONSTRAINTS =
+            "TODO CONSTRAINTS";
+
+    /*
+     * TODO VALIDATION REGEX
+     */
+    //public static final String VALIDATION_REGEX = "[\\p{Alnum}][\\p{Alnum} ]*";
+
+    public final String location;
+
+    /**
+     * Constructs a {@code Location}.
+     *
+     * @param location A valid location.
+     */
+    public Location(String location) {
+        requireNonNull(location);
+        checkArgument(isValidLocation(location), MESSAGE_CONSTRAINTS);
+        this.location = location;
+    }
+
+    /**
+     * Returns true if a given string is a valid platform.
```

Comment:
Platform also needs to be changed to location here

Reply:
fixed in 3549fd3